spec/features/content_item_signup_spec.rb

```diff
@@ -50,9 +50,7 @@ def when_i_visit_the_signup_page
   end
 
   def and_i_refine_my_selection
-    expect(page).to have_checked_field("topic-2")
```

Comment:
is it worth asserting that we have no checked fields?

Reply:
Oh yes, good point - I missed the controller spec doesn't actually cover the lack of pre-selection in the UI. I've amended the commit to negate this check.